lib/doorkeeper/config.rb

```diff
@@ -228,33 +228,24 @@ def realm
       @realm ||= 'Doorkeeper'
     end
 
-    def authorization_response_types
-      @authorization_response_types ||= calculate_authorization_response_types
+    def enabled_grant_flows
+      @enabled_grant_flows ||= grant_flows.map{ |name| GrantFlow.get(name) }
```

Comment:
It would be nice to have grant_flows here be an array of Flow objects, but I don't want to break compatibility with existing configurations.